### DISCUSSION TITLE
Displaying high resolution content while using lower resolution content for detection

### DISCUSSION THREAD

**Discussion by marcgarciamarti** [blakeblackshear/frigate#8139] [Ask A Question] [answered]
Hello,

I have frigate up and running. It is a lovely piece of software. I'm very very impressed.

I was wondering if there is anyway on Home Assistant to get the highest resolution feed displayed (the one used for recording purposes) rather than the lower resolution feed, which seems to get picked up by default?

thanks

**Comment by NickM-27** [comment, marked as answer]
Yes, just setup go2rtc to send the high res stream by making the name of the high res stream and the camera match 

https://docs.frigate.video/guides/configuring_go2rtc

**Comment by marcgarciamarti** [comment]
> Yes, just setup go2rtc to send the high res stream by making the name of the high res stream and the camera match
> 
> https://docs.frigate.video/guides/configuring_go2rtc

ok! thanks! I had thought about this too but configuring go2rtc seemed a bit complex, hence I thought I could post this here first. Thanks!
